sonar-project.properties

```
@@ -3,5 +3,6 @@ sonar.organization=moveit
 
 sonar.python.version=3
 sonar.sources=.work/target_ws/src
+sonar.cfamily.llvm-cov.reportPath=.work/target_ws/coverage.info
 
```

just for now can we see if disabling SCM exclusion importing fixes the report? then we can debug why it thinks everything is gitignored after we have a successful scan.

```suggestion
sonar.scm.exclusions.disabled=true
```